backend/src/v4/config.js

```diff
@@ -279,7 +279,7 @@ config.fileUploads = {
 	modelSizeLimit: config.uploadSizeLimit,
```

Comment:
Empty image placeholder (icon) is not centred
![image](https://github.com/3drepo/3drepo.io/assets/43547703/1a39989f-228b-4c3c-8b4e-ed32dcca739f)